.travis.yml

```diff
@@ -5,6 +5,7 @@ go:
 
 env:
   - DEP_VERSION="0.5.2"
+  - OPERATOR_SDK_VERSION="v0.8.1"
```

Comment:
It looks like the `env` describes one-build-per-line. So this might have to be like this instead:

```yaml
env:
  - DEP_VERSION="0.5.2" OPERATOR_SDK_VERSION="v0.8.1"
```

That's an... odd... choice by Travis, but I understand why it's necessary. I wish there was a better way to check that this sort of thing would happen _before_ pushing.